content/60.methods.md

```diff
@@ -34,14 +34,14 @@ COVID-19, open publishing, open-source, data integration, manubot
 
 Coronavirus Disease 2019 (COVID-19) has shaped the years 2020 and 2021 by causing a worldwide public health crisis.
 The scientific community has responded by turning significant attention and resources towards COVID-19 and the associated virus, SARS-CoV-2.
-The result has been the rapid release of data, results, and publications related to COVID-19 at a scale never previously seen.
+The result has been the rapid release of data, results, and publications at a scale never previously seen for a single topic.
 Over 20,000 articles about COVID-19 were released in the first four months of the pandemic [@doi:10.1053/j.ackd.2020.08.003], and the velocity and volume of information being released led to the pandemic being termed an "infodemic" as well [@doi:10.1053/j.ackd.2020.08.003; @doi:10/ggpx67].
 While this influx of information is likely evidence of important work towards understanding the virus and the disease, there are also downsides to the availability of too much information.
-The downsides of "excessive publication" have been recognized for over forty years, as it was raised as a major concern about the move towards electronic, rather than print, publishing at the turn of the millennium [@doi:10/d3bmnv].
+The downsides of "excessive publication" have been recognized for over forty years, and it was raised as a major concern about the move towards electronic, rather than print, publishing at the turn of the millennium [@doi:10/d3bmnv].
 The contents of the COVID-19 Open Research Dataset (CORD-19) [@arxiv:2004.10706], which was developed in part to assist in efforts to train machine learning algorithms on COVID-19-related text, illustrates the volume of scholarly literature relevant to understanding this virus (Figure @fig:cord19-growth).
-This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated viruses [@arxiv:2004.10706].
+This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated diseases [@arxiv:2004.10706].
```

Comment:
I recall in the prior reviews there was a comment about CORD-19 not being just a dataset.  We should address that if possible (not necessarily here).

Reply:
I am a bit confused by this comment because everything I has seen does refer to CORD-19 as a dataset, although it's a dataset specifically assembled for use in AI -- for example, the link the reviewer provided says this: "The COVID-19 Open Research Dataset (CORD-19) is a growing resource of scientific papers on COVID-19 and related historical coronavirus research."

Our introduction of CORD-19 says: "The contents of the COVID-19 Open Research Dataset (CORD-19) [@arxiv:2004.10706], which was developed in part to assist in efforts to train machine learning algorithms on COVID-19-related text, illustrates the volume of scholarly literature relevant to understanding this virus (Figure @fig:cord19-growth)."

I'm not totally clear on how this is different than what the reviewer was suggesting!

Reply:
I went back to the reviewer comment and am also confused now.  I think our text is clear enough.  Let's leave what we have.

Reply:
OK I'm glad it's not just me! Thank you so much Tony.